### PR TITLE
Usage of default value in ${…​} enclosure

### PR DESCRIPTION
**Pull request by tlappe**
You can set a default value in keycloak.json with ${...} but documentation is missing.

### CODE REVIEW

**Comment by stianst** [comment]
@andymunro from a technical perspective this looks good, but can you review it as well?

**Review by stianst — APPROVED** [review]

File: securing_apps/topics/oidc/java/java-adapter-config.adoc [hunk 1]
```diff
@@ -44,7 +44,7 @@ This is what one might look like:
 ----
 
 You can use `${...}` enclosure for system property replacement. For example `${jboss.server.config.dir}` would be replaced by `/path/to/{project_name}`.
-Replacement of environment variables is also supported via the `env` prefix, e.g. `${env.MY_ENVIRONMENT_VARIABLE}`.
+Replacement of environment variables is also supported via the `env` prefix, e.g. `${env.MY_ENVIRONMENT_VARIABLE}`. You can also define a default value `${...:default_value}` which is used if the system property `${jboss.server.config.dir:default_value}` or the environment variable `${env.MY_ENVIRONMENT_VARIABLE:default_value}` is absent.
```
Comment (andymunro) [inline securing_apps/topics/oidc/java/java-adapter-config.adoc:47]: Add a comma before which.

**Review by andymunro — CHANGES_REQUESTED** [review]
One change needed

**Comment by andymunro** [comment]
@tlappe To fix the Waiting for Status problem, use this command: git pull upstream main

**Comment by ahus1** [comment]
As this PR has been abandoned, this is now superseded by #1795

**Pull request closed without merging**